### PR TITLE
The last cards from the first set of Moon cards. (With two exceptions.)

### PR DESCRIPTION
**Pull request by kberg**

### CODE REVIEW

File: src/cards/moon/MooncrateBlockFactory.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    super({
+      name: CardName.MOONCRATE_BLOCK_FACTORY,
+      cardType: CardType.ACTIVE,
+      tags: [Tags.BUILDING],
+      cost: 8,
+
+      metadata: {
+        description: 'Effect: When you pay for a luna standard project (colony, road, mine), you spend 4MC less. / Requires 1 mine on the Moon.',
+        cardNumber: 'M38',
+        renderData: CardRenderer.builder((_b) => {}),
```
Comment (chosta) [inline src/cards/moon/MooncrateBlockFactory.ts:21]: `renderData` is optional, you could probably skip this line

Comment (chosta) [reply src/cards/moon/MooncrateBlockFactory.ts:21]: Is this mssing the effect actually?

**Review by chosta — APPROVED** [review]
LGTM. I'd like to also check the visuals after the merge.

File: src/cards/moon/MooncrateBlockFactory.ts (hunk 1 above)
Comment (kberg) [reply src/cards/moon/MooncrateBlockFactory.ts:21]: It probably is. There is an absolute need for the cards to get a viz polish.

Comment (chosta) [reply src/cards/moon/MooncrateBlockFactory.ts:21]: I'll check them once you get them all up.

Comment (kberg) [reply src/cards/moon/MooncrateBlockFactory.ts:21]: Thanks. Me too - it'll just be good to do them at once.

**Pull request merged**